app/assets/javascripts/views/vacation_request.js

```diff
@@ -0,0 +1,123 @@
+App.Views.VacationRequest = Backbone.View.extend({
+  el: 'section',
+  template: JST['templates/vacation_request'],
+
+  initialize: function() {
+    this.$el.html( this.template() );
+    // Prepare handy access for the controls
+    this.$available_days  = this.$('#available_days');
+    this.$vacation_type   = this.$('select[name=vacation_type]');
+
+    this.available_vacations  = new App.Collections.AvailableVacations();
+    this.vacation_requests    = new App.Collections.VacationRequests();
+    this.listenTo( this.available_vacations,  'reset',  this.render );
+    this.listenTo( this.vacation_requests,    'all',  this.logg );
+    // this.listenTo( this.collection, 'sync', this.render );
+    this.available_vacations.fetch({reset: true});
+    this.vacation_requests.fetch({reset: true});
+    // Set date picker variables
+    this.startDate  = new Date();
+    this.endDate    = this.startDate;
+    this.duration = 0;
+  },
+
+  render: function() {
+    // console.log(JSON.stringify(this.available_vacations));
+    // console.log(JSON.stringify(this.vacation_requests));
+    this.updateAvailableDays();
+    this.updateOkButton();
+
+    // TODO: Disable option as described below when there is no days available
+    // http://www.mkyong.com/jquery/how-to-set-a-dropdown-box-value-in-jquery/
+
+    return this;
+  },
+
+  events: {
+    'click  .ok':                         'onCreate',
+    'click  .cancel':                     'onCancel',
+    'change select[name=vacation_type]':  'onTypeChange',
+    'change input[name=start]':           'onStartDate',
+    'change input[name=end]':             'onEndDate',
+  },
+
+  calculateDuration: function( a, b ) {
```

Comment:
Please give  to a, b parameters more meaningful names, for example 'start', 'end'.